clang/unittests/Tooling/Syntax/TokensTest.cpp

```diff
@@ -585,6 +585,17 @@ TEST_F(TokenCollectorTest, DelayedParsing) {
   EXPECT_THAT(collectAndDump(Code), StartsWith(ExpectedTokens));
 }
 
+TEST_F(TokenCollectorTest, UnclosedToken) {
```

Comment:
I started looking at this to try and make some progress towards getting this landed. However, when running this test locally, I'm not seeing either of the branches added to `spelledForExpandedToken()` being taken.

Am I missing something, or is this test not exercising the intended codepath?

Reply:
Indeed, it looks like `TokenBuffer::spelledForExpanded()` (which is the only call site of `spelledForExpandedToken()`)  is not called by anything currently in this test.

To exercise `spelledForExpanded()`, we will need to make an explicit call to it, the way a number of other tests in this file do.